Submissions/uglyprincess/task_1/task1.py

```diff
@@ -0,0 +1,19 @@
+import csv, json 
+csvFilePath = 'C:/Users/Aaryan/Desktop/Monty_Python_Class/Task_1/movies.csv'
+jsonFilePath = 'movies.json'
+
+data = {}
+
+with open(csvFilePath, encoding = 'utf-8') as csvFile:
+	csvReader = csv.DictReader(csvFile)
+	print(csvReader)
+	for csvflow in csvReader:
+		movieId = csvflow["movieId"]
+		data[movieId] = csvflow
```

Comment:
You have to create the json file containing only comedy movies.

Reply:
Okay sir